radiomesh/scripts/gen_expr.py

```diff
@@ -44,10 +44,10 @@ def sympy_expressions(
         from gains and weights.
   """
   # set up symbolic expressions
-  gp00, gp10, gp01, gp11 = sympy.symbols(" ".join(P_GAIN_ARGUMENTS), real=False)
-  gq00, gq10, gq01, gq11 = sympy.symbols(" ".join(Q_GAIN_ARGUMENTS), real=False)
+  gp00, gp01, gp10, gp11 = sympy.symbols(" ".join(P_GAIN_ARGUMENTS), real=False)
+  gq00, gq01, gq10, gq11 = sympy.symbols(" ".join(Q_GAIN_ARGUMENTS), real=False)
   w00, w01, w10, w11 = sympy.symbols(" ".join(WEIGHT_ARGUMENTS), real=True)
-  v00, v10, v01, v11 = sympy.symbols(" ".join(VIS_ARGUMENTS), real=False)
+  v00, v01, v10, v11 = sympy.symbols(" ".join(VIS_ARGUMENTS), real=False)
```

Comment:
@landmanbester The mistake was here as the various argument constants did not align with the tuple destructuring.

The numba wgridder and `explicit_gridder` now match on my local setup, and hopefully the github action CI will confirm this.

Reply:
Yeah, I assumed row based indexing to match the MS convention

Reply:
I don't think I spotted that and assumed monotonicity on autopilot mode.